clients/v1_shell/v1_shell.go

```diff
@@ -51,7 +52,7 @@ type PersistentQueryParams struct {
 
 func DefaultPersistentQueryParams() PersistentQueryParams {
 	return PersistentQueryParams{
-		Format:       ColumnFormat,
+		Format:       TableFormat,
```

Comment:
Do we have a `-execute` flag yet? When/if we add one (to run in a script) we should make `ColumnFormat` the default for that.

Reply:
No flag yet, I can create an issue for it.